src/Modal.ts

```diff
@@ -128,8 +128,11 @@ class CameraModal extends Modal {
 
 
     switchCameraButton.onclick = async () => {
-      cameraIndex = (cameraIndex + 1) % cameras.length;
-      this.videoStream = await getVideoStream();
+    	cameraIndex = (cameraIndex + 1) % cameras.length;
+		this.videoStream = null;
```

Comment:
Thank You for the Pull Request.

We do not need this line to set videoStream to null right? 
Kindly remove it and we're good to merge!

Reply:
You are correct. I believe I have removed it successfully.